src/components/AddToSigner.tsx

```diff
@@ -19,8 +18,10 @@ export default function AddToSigner({ path }: QrInfo) {
     <>
       <div className="flex items-center mt-5">
         <Button
-            onClick={openModal}
-            label="Add to Signer"
+          className="w-full"
+          backgroundColor={color}
+          onClick={openModal}
+          label="Add to Signer"
```

Comment:
Phone logo is missing
<img width="602" alt="image" src="https://user-images.githubusercontent.com/7165377/172148638-fa854739-ca84-45a9-ab9e-5b2635aec557.png">
https://www.figma.com/file/eOnz3xbzObTG8hQlIRs6jg/Metadata-Portal?node-id=0%3A1